### ISSUE TITLE
[BUG] Music started playing through bluetooth in car even though app wasn't open

### ISSUE THREAD

**Issue by RPNate1** [BillGoldsmith/iOS-bugfest#118]
## Brief description of the issue: 
I had just closed all apps. Got in my car and was confused by music playing after a minute of driving. Checked to see if any apps were open and RP didn't show as open (foreground or background). Clicked on MK2 and it opened, already playing (music continued without interruption). 

### Steps to reproduce:
Close app. 
Turn on Bluetooth device. 
See if app starts playing music.

### Expected result:
Music should. not play if app wasn't already open.

### Actual result:
App started playing music and did  not show as open anywhere.

### Where did you see it? (App build version, device type and iOS version):
b55, iPhone 11 pro, iOS 16.4

*(Please provide screenshots or error message text when possible.)*

**Comment by bradtchapman** [comment]
@RPNate1 @donshefer - you are misunderstanding the behavior here, Nate.

Don designed this according to Apple's intent.  It is 100% expected that reconnecting to a car with bluetooth may trigger a "Play" action, which would cause the iphone to resume whatever was last playing.  In this case, RP.

The RP app is allowed to sleep indefinitely just like all apps, and if something calls for media playback, it will play.

Did you forget to turn down your stereo after a vigorous drive, and get blasted with music at full volume ?

**Comment by donshefer** [comment]
@RPNate1 I did some testing around this and was not able to replicate this behavior.
The app correctly pauses audio when disconnecting a bluetooth device. And correctly does *not* restart play when reconnecting the same or a new bluetooth device. I tested against the Music and Spotify apps for reference and we are replicating their behavior.
I also tested playback when closing the app entirely. Playback was stopped as expected and didn't come back until the app was relaunched.
I'm not sure what you saw, but I will keep an eye out for it.
